docs/migration/migrate_6_0.asciidoc

```diff
@@ -29,3 +29,9 @@ This is no longer the case. Now, only commas are a valid query separator: e.g. `
 
 
 *Impact:* You will need to migrate your stored timelion queries to the new syntax.
+
+[float]
+=== Removed same port http to https redirect behavior
+*Details:* Kibana 5.x redirected requests from http to https on the same port if TLS was configured.  This was removed.
```

Comment:
Rather than "This was removed", let's err on the side of more information even if it seems a bit redundant.  Perhaps "Kibana 5.x redirected requests from http to https on the same port if TLS was configured. Starting in Kibana 6.0.0, Kibana no longer redirects basic http traffic to https."